### DISCUSSION TITLE
Publish log-viewer on another port

### DISCUSSION THREAD

**Discussion by pitah81** [opcodesio/log-viewer#228] [Q&A]
I need to publish log-viewer on a different port as to be able to limit access to logs through central firewall whitelist rules. Is this possible somehow, and how would I accomplish moving just this service to another port for all instanses of log-viewer on this server?

**Comment by arukompas** [comment]
@pitah81 , since this is a Laravel **package**, it's only accessible through a Laravel application itself.

I guess you could create a new Laravel app, run it on a different port, and configure `config/log-viewer.php` to include logs from various other paths on the disk.
